src/jit/lowerxarch.cpp

```diff
@@ -2808,6 +2809,24 @@ void Lowering::ContainCheckHWIntrinsic(GenTreeHWIntrinsic* node)
                             break;
                         }
 
+                        case NI_AVX2_Permute4x64:
+                        {
+                            // These intrinsic have op2 as an imm and op1 as a reg / mem
+
+                            if (HWIntrinsicInfo::isImmOp(intrinsicId, op2))
```

Comment:
`Permute4x64` only has IMM form, you do not need this check.

Reply:
Yeah, already fixed in my code 😃